lib/MetaCPAN/Script/Contributor.pm

```diff
@@ -0,0 +1,118 @@
+package MetaCPAN::Script::Contributor;
+
+use strict;
+use warnings;
+
+BEGIN {
+    $ENV{PERL_JSON_BACKEND} = 'JSON::XS';
```

Comment:
oops, here it was copy-pasted from Script::Release... there it was added in 2f6d07b5e2... not sure how relevant it is.
i'll remove it from this script.